sube/src/lib.rs

```diff
@@ -405,11 +411,19 @@ pub struct StorageChangeSet {
 /// }
 /// ```
 pub trait Backend {
-    async fn query_storage_at(
+    async fn get_storage_items(
         &self,
         keys: Vec<String>,
         block: Option<String>,
```

Comment:
I think we should have this only as a number(e.g. `u32`) and do resolution internally in the RPC implementations . 99% of the time users want to use a block number instead of a hash. If we were to use a hash we should also accept it as a byte array instead of string since the hex encoding of stuff is very JSON RPC specific.